.github/workflows/stable-snapshot-timestamp.yml

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Stable Snapshot and Timestamp
+
+permissions: read-all
+
+# Execute this as a biweekly cron job and on changes to repository/
+# when new published metadata is submitted.
+on:
+  # Enable cron for re-signing snapshot and timestamp every week
+  schedule:
+    - cron: '0 0 */7 * *' # every 7 days
+  # When a new root is staged
+  push:
+    branches:
+      - main
+    paths:
+      - 'repository/staged/root.json'
+  workflow_dispatch:
+
+jobs:
+  run_snapshot_timestamp_publish:
+    runs-on: ubuntu-20.04
+    permissions:
+      id-token: 'write'
+      issues: 'write'
+      pull-requests: 'write'
+      contents: 'write'
+    steps:
+      - uses: sigstore/root-signing/.github/workflows/reuseable-snapshot-timestamp.yml@main
+        with:
+          snapshot_key: 'gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot'
+          timestamp_key: 'gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/timestamp'
+          repo: 'repository/'
+          provider: projects/237800849078/locations/global/workloadIdentityPools/root-signing-pool/providers/sigstore-root'
+          service_account: 'sigstore-root-signing@project-rekor.iam.gserviceaccount.com'
```

Comment:
Actually, can you explain why these values are all different from the old ones?

Reply:
the v4 root uses these keys (see the old history, before 2 weeks ago, or the successful runs of this that are scheduled). the v5 root rotated the keys so we can move them into a separate GCP project where on-call'ers and maintainers don't have perms. 

`stable` refers to the v4 stable root, i'll make a separate workflow for the v5 `beta` root with the different values.